sdk/deviceupdate/Azure.IoT.DeviceUpdate/README.md

```diff
@@ -22,7 +22,7 @@ For the best development experience, developers should use the official Microsof
 Install the Device Update for IoT Hub client library for .NET with [NuGet](https://www.nuget.org/ ):
 
 ```dotnetcli
-dotnet add package Azure.IoT.DeviceUpdate --version 1.0.0-beta.2
+dotnet add package Azure.IoT.DeviceUpdate --version 1.0.0-beta.1
```

Comment:
Don't include version, as it would need to be updated every release.

Reply:
https://www.nuget.org/packages/Azure.Iot.DeviceUpdate/1.0.0-beta.2

1.0.0-beta.2 is also a released version.  Our tooling originally had updated this to 1.0.0-beta.3 -- what motivating changing this?

Reply:
updated